### ISSUE TITLE
Map doesn't load due to "ReferenceError: Plotly is not defined"

### ISSUE THREAD

**Issue by Sirs0ri** [flobz/psa_car_controller#649]
_Edit: if you're also experiencing this bug, there's a workaround described [here](https://github.com/flobz/psa_car_controller/issues/649#issuecomment-1793506080)!_

**Describe the bug**

When opening the Dashboard at http://<ip>:5000, I can't access the map, the tab changes to "active", but it won't have any content.
Looking at my browser's console I see endless errors about `ReferenceError: Plotly is not defined`, together with a massive spam of POST requests to http://<ip>:5000/_dash-update-component (about 10x per second), loading a gigabyte of data within minutes and ultimately DDoSing the controller.

The error seems to originate from `webpack:///src/fragments/Graph.react.js`

This bug started happening with the update to docker tag 3.0.13

I'll attach a log below, let me know if there's any other info you need!

**To Reproduce**
1. Run the controller in docker, from `flobz/psa_car_controller:v3.0.12`
2. open the dashboard in the browser
3. open the map - it won't load
4. open the browser's console, and observe the flood of warnings
5. open the devtools' network connections tab, and observe the flood of POS requests

**Config file**

`config.ini`
```ini
[General]
currency = €
# minimum trip length in km so it's added to stats and map in website
minimum trip length = 10
# for future use
length unit = km
[Electricity config]
# price by kw/h
day price = 0.2499
night price
# ex: 22h30
night hour start
# ex: 6h00
night hour end
dc charge price
high speed dc charge price
# minimum power in kW that should be delivered during a charge so it can be considered as a high speed charger
high speed dc charge threshold
charger efficiency = 0.8942
```

`config.json`
```json
{
    "abrp": {
        "abrp_enable_vin": [],
        "token": ""
    },
    "client_id": "REDACTED",
    "client_secret": "REDACTED",
    "co2_signal_api": null,
    "country_code": "DE",
    "customer_id": "REDACTED",
    "proxies": {
        "http": "",
        "https": ""
    },
    "realm": "clientsB2CPeugeot",
    "refresh_token": "REDACTED",
    "remote_refresh_token": "REDACTED",
    "weather_api": "REDACTED"
}
```

**Environment (please complete the following information):**
 - OS: Docker, `flobz/psa_car_controller:v3.0.12`
 - Brand and model of car: Peugeot e208
 - Application version (commit number, branch or tag) 
 - Log from http://server_address:port/log and browser console log

**Additional context**

`activity.log` (first 100 lines)
```
2023-11-01 17:49:01,449 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:01] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:01,542 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:01] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:01,636 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:01] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:01,730 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:01] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:01,822 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:01] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:01,914 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:01] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:02,049 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:02] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:02,140 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:02] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:02,230 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:02] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:02,320 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:02] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:02,408 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:02] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:02,498 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:02] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:02,591 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:02] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:02,680 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:02] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:02,771 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:02] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:02,876 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:02] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:02,972 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:02] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:03,070 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:03] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:03,165 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:03] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:03,261 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:03] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:03,354 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:03] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:03,446 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:03] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:03,539 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:03] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:03,635 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:03] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:03,732 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:03] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:03,830 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:03] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:03,931 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:03] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:04,087 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:04] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:04,178 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:04] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:04,267 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:04] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:04,355 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:04] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:04,443 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:04] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:04,533 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:04] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:04,621 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:04] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:04,709 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:04] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:04,798 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:04] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:04,892 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:04] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:04,985 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:04] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:05,082 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:05] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:05,175 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:05] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:05,268 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:05] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:05,357 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:05] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:05,446 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:05] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:05,548 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:05] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:05,642 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:05] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:05,731 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:05] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:05,824 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:05] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:05,917 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:05] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:06,008 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:06] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:06,098 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:06] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:06,188 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:06] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:06,283 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:06] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:06,375 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:06] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:06,463 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:06] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:06,553 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:06] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:06,641 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:06] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:06,730 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:06] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:06,818 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:06] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:06,909 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:06] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:06,999 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:06] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:07,091 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:07] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:07,180 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:07] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:07,282 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:07] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:07,399 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:07] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:07,495 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:07] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:07,586 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:07] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:07,685 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:07] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:07,797 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:07] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:07,886 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:07] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:07,975 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:07] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:08,063 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:08] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:08,154 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:08] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:08,303 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:08] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:08,398 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:08] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:08,489 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:08] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:08,641 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:08] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:08,736 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:08] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:08,826 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:08] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:08,922 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:08] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:09,019 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:09] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:09,110 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:09] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:09,201 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:09] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:09,316 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:09] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:09,404 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:09] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:09,502 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:09] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:09,590 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:09] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:09,700 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:09] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:09,798 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:09] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:09,888 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:09] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:09,980 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:09] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:10,070 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:10] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:10,163 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:10] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:10,254 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:10] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:10,346 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:10] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:10,442 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:10] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:10,536 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:10] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:10,624 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:10] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:10,713 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:10] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:10,804 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:10] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
2023-11-01 17:49:10,911 :: INFO :: MY_IP_REDACTED - - [01/Nov/2023 17:49:10] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
```

**Comment by kvikindi** [comment]
Same issue here, more or less identical, except in Home Assistant add-on.
Coincidentally also not seeing anything in the "Control" tab, but not sure if it's related there.

**Comment by SabatoArdolino** [comment]
I've the same issue. I've installed the psa car controller by HA Addon and in the v3.0.13 the MAP TAB is empty. I can see the other tab with old data. Any idea ?

**Comment by flobz** [comment]
Sorry I don't have this issue, so I can't debug.

**Comment by SabatoArdolino** [comment]
I've opened a new issue for this but you wrote "duplicated" ... How can I give you all information ?

**Comment by bbr111** [comment]
Same here.
Title is Updating...
Map is white site. complete empty.

**Comment by bbr111** [comment]
https://github.com/flobz/psa_car_controller/blob/d759102c4a03e7f66bfcab17fb44637d9104cc3a/psa_car_controller/web/app.py#L60C1-L71C52

I think the error is here.

**Comment by bbr111** [comment]
psa_car_controller/web/assets
should there be a file plotly-with-meta.js ?
if i copy it to the docker container and restart the app.
everything is working.

**Comment by SabatoArdolino** [comment]
> psa_car_controller/web/assets should there be a file plotly-with-meta.js ? if i copy it to the docker container and restart the app. everything is working.

Could you share the file and / or the solution ? I've checked in my folder and I con't find the file plotly-with-meta.js

**Comment by bbr111** [comment]
wget https://raw.githubusercontent.com/plotly/plotly.js/master/dist/plotly-with-meta.js
docker cp plotly-with-meta.js psa_car_controller1:./usr/local/lib/python3.9/dist-packages/psa_car_controller/web/assets/

replace psa_car_controller1 with your docker containername

I don't know if this is the right way, but it solves the map problem for me.

**Comment by SabatoArdolino** [comment]
Hi and thanks for reply. I've installed the addon via hacs and the procedure is little different but work. I've downloaded the file and put it into the 3 folders that I found in my Virtual Machine. After that the maps work but I've another problem because I see the trip how I "fly" fron point a to point 2.
![2023-11-05_061429](https://github.com/flobz/psa_car_controller/assets/137244513/688da1a2-294b-4867-bf2a-850b4d19c97a)

**Comment by Sirs0ri** [comment]
The workaround described by @bbr111 [here](https://github.com/flobz/psa_car_controller/issues/649#issuecomment-1793506080) works for me.

> Sorry I don't have this issue, so I can't debug.

@flobz is there anything we can help with?

**Comment by Mikado8231** [comment]
@Sirs0ri @bbr111 any chances to test my PR #652? (without manual assets) 

_I have no data for the trips map_.

**Comment by bbr111** [comment]
@Mikado8231 internal server error 500

**Comment by Mikado8231** [comment]
@bbr111 have you reset your image to retrieve v3.0.13?

Have you a log?

**Comment by sod75** [comment]
Maybe this helps someone to track down what the underlying issue is... 
I never did anything with the plotly-with-meta.js  file

- For me the map was not working on docker:latest (v3.0.10) before
- After updating to  Docker image v3.0.12 it was working fine after deleting the info.db ( trips fly from start to end though) 
 
- I just tried updating to v3.0.13 mentioned in this thread and the map stopped working.
- I pulled v3.0.12 again, but the issue persists. (trips, etc are there, map is blank, tab hangs on  "updating)

Happy to try something out to help troubleshoot

**Comment by Sirs0ri** [comment]
This seems to be fixed in 3.1.0, thanks!
